source/renderer/app/features/discreet-mode/ui/discreet-toggle-top-bar/DiscreetToggleTopBar.js

```diff
@@ -16,27 +16,39 @@ type Props = {
 };
 
 const DiscreetToggleTopBar = ({ intl, hasTadaIcon }: Props) => {
-  const discreetModeFeature = useDiscreetModeFeature();
+  const {
+    isDiscreetMode,
+    isSettingsTooltipEnabled,
+    toggleDiscreetMode,
+    setDiscreetModeSettingsTooltip,
+  } = useDiscreetModeFeature();
+  const [visible, setVisible] = useState(false);
 
   return (
-    <div className={classnames(styles.root, hasTadaIcon && styles.hasTadaIcon)}>
+    <div
+      className={classnames(styles.root, hasTadaIcon && styles.hasTadaIcon)}
+      onMouseEnter={() => setVisible(true)}
+      onMouseLeave={() => setVisible(false)}
+    >
       <PopOver
+        visible={visible || isSettingsTooltipEnabled}
```

Comment:
When visible prop is used, we cannot go back to an automatic hover effect due to a bug in the tippy library. This is why it was necessary to use a visible state.

https://github.com/atomiks/tippyjs-react/issues/325